ts/parts/Axis.ts

```diff
@@ -5364,8 +5382,14 @@ extend(Axis.prototype, /** @lends Highcharts.Axis.prototype */{
             tickPositionsOption = options.tickPositions,
             minorTickIntervalOption = this.getMinorTickInterval(),
             tickPositioner = options.tickPositioner,
-            startOnTick = options.startOnTick,
-            endOnTick = options.endOnTick;
+            hasVerticalPanning = this.hasVerticalPanning(),
+            isColorAxis = this.coll === 'colorAxis',
+            startOnTick = isColorAxis ?
```

Comment:
More compact:
`startOnTick = !hasVerticalPanning && !isColorAxis && options.startOnTick,`

Reply:
Unfortunately, this does not work as expected, so I changed it to:
`startOnTick = (isColorAxis || !hasVerticalPanning) && options.startOnTick`